tests/integration/targets/ios_ospfv2/tests/cli/_populate_config.yaml

```diff
@@ -27,9 +27,9 @@
                 - name: test_prefix_out
                   direction: out
           network:
-            address: 198.51.100.0
-            wildcard_bits: 0.0.0.255
-            area: 5
+            - address: 198.51.100.0
```

Comment:
Can we have tests with multiple networks, i.e., test that lists of dicts work?

Reply:
@NilashishC yes, have added the respective TC scenario to [unit TC](https://github.com/ansible-collections/cisco.ios/pull/152/files#diff-7fc64dfe232b95f2b079ce5b4664ff12R90) instead of integration to keep it lightweight and to test for any new features that we add.